### ISSUE TITLE
Make rollover images flush to the edges

### ISSUE THREAD

**Issue by NewLabCode** [jakec729/matheiken-redesign#19]
Can you make the rollover images have no padding for the top alignments, so that they appear flush to the edges on the top right, top left? You have it that way for bottom alignment, I want the same for top.

**Comment by jakec729** [comment]
If we do that we will cover up that navigation, is that prefered?

**Comment by NewLabCode** [comment]
Yes it's only for rollover and when u move towards nav rollover goes away anyway, so it's ok.

**Comment by jakec729** [comment]
Tried to make a tweak to figure this out, but because the nav is fixed to the top with a white background to cover up what's beneath, we can't have the hover image simultaneously under the text and over the nav (this is the case with public-library as well).
